### DISCUSSION TITLE
i am a new GitHub  education student. how i can use this platform effectively

### DISCUSSION THREAD

**Discussion by muhammadShahidk** [community/community#56948] [GitHub Education] [answered]
### Hi everyone,

Subject: Seeking Help as a New GitHub Education Student

Dear [Friend]

I hope this message finds you well. As a new student on GitHub Education, I'm reaching out to ask for your assistance. I am eager to learn and explore the platform, but I could use some guidance from someone experienced like you.

Being new to GitHub, I'm still getting familiar with its features and how to use it effectively for version control, collaboration, and showcasing my projects. I would greatly appreciate your support in helping me understand the basics, sharing best practices, and answering any questions that may arise.

If you have some time to spare, I would be grateful for a virtual meeting or even a chat to discuss GitHub Education and gather your insights. Additionally, if you know of any helpful resources or tutorials, I would love to have access to them.

Your assistance will make a significant impact on my learning journey and enable me to make the most of GitHub Education. I genuinely value your mentorship, and I'm excited about the knowledge and experiences that lie ahead.

Thank you for considering my request, and I look forward to your support.

**Comment by samus-aran** [comment, marked as answer]
Hi @muhammadShahidk 👋 

Once you become a verified student you have access to [Global Campus](https://education.github.com/globalcampus/student). 

GitHub Global Campus is a portal from which you can access your GitHub Education benefits and resources, all in one place. On the GitHub Global Campus portal, students can:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

Hope this helps 👍

**Reply by muhammadShahidk** [reply]
thanks for your knowledgeable response @samus-aran

**Comment by RyzenFromFire** [comment]
You may find the following resources helpful
- https://github.com/jfiksel/github-classroom-for-students
- https://www.youtube.com/watch?v=jXpT8eOzzCM&list=PL7uPCUbavAWf7cEh0R1IhGaX_FUMD01Fr&index=2

I'm also working on a comprehensive tutorial for both instructor and student perspective, so keep an eye out for that if you like.

**Reply by samus-aran** [reply]
Oh wow @RyzenFromFire. These are awesome. I'm sure this will be extremely helpful for the student community! 🚀

**Reply by RyzenFromFire** [reply]
Thanks! I just published the [aforementioned tutorial](https://github.com/rzn-example-classroom/tutorial) earlier.

**Comment by muhammadShahidk** [comment]
Thank you for your response. I will try to get most of it
